package/com.unity.formats.usd/Tests/Editor/Unity.Formats.USD.Tests.Editor.asmdef

```diff
@@ -2,16 +2,25 @@
     "name": "Unity.Formats.USD.Tests.Editor",
     "references": [
         "Unity.Formats.USD.Runtime",
-        "Unity.Formats.USD.Editor"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Formats.USD.Editor",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
     "includePlatforms": [
         "Editor"
     ],
+    "excludePlatforms": [],
     "allowUnsafeCode": true,
+    "overrideReferences": true,
     "precompiledReferences": [
-        "USD.NET.dll"
-    ]
+        "USD.NET.dll",
+        "USD.NET.Unity.dll",
```

Comment:
I actually only add this dll. Not sure why there is so much that changed. I guess assemblies serialization changed.

Reply:
No worries, I reverted the unwanted changes.